content/operate/oss_and_stack/stack-with-enterprise/search/query-performance-factor.md

```diff
@@ -1,17 +1,17 @@
 ---
-Title: Enable scalable Redis Query Engine in Redis Enterprise
+Title: Enable Query Performance Factor For Redis Query Engine in Redis Enterprise
 alwaysopen: false
 categories:
 - docs
 - operate
 - stack
-description: Enable the scalable Redis Query Engine in Redis Enterprise to increase the performance of queries.
+description: Enable the Query performance factor for Redis Query Engine in Redis Enterprise to increase the performance of queries.
 linkTitle: Enable scalable Redis Query Engine
 weight: 20
-aliases: /operate/oss_and_stack/stack-with-enterprise/search/scalable-search/
+aliases: /operate/oss_and_stack/stack-with-enterprise/search/query-performance-factor/
```

Comment:
We need to keep the old alias to avoid possible broken links from external sources.
```suggestion
aliases: /operate/oss_and_stack/stack-with-enterprise/search/scalable-search/
         /operate/oss_and_stack/stack-with-enterprise/search/query-performance-factor/
```